plugins/20_events/commands

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
+source "$(dirname $0)/../common/functions"
+
+find "$l_plugin_dir" -type l -printf "%f " | sort
+
+case "$1" in
+  events)
+    if [[ -f $DOKKU_EVENTS_LOGFILE ]] ; then
+      if [[ $2 == "-t" ]]; then
+        tail -f $DOKKU_EVENTS_LOGFILE
+      else
+        tail -n 100 $DOKKU_EVENTS_LOGFILE
+      fi
+    fi
+    ;;
+
+  events:on)
+    echo "Enabling dokku events logger"
+    [[ -d $DOKKU_ROOT/.dokkurc ]] || mkdir -p $DOKKU_ROOT/.dokkurc
+    echo "export DOKKU_EVENTS=1" > $DOKKU_ROOT/.dokkurc/DOKKU_EVENTS
+    ;;
+
+  events:off)
+    echo "Disabling dokku events logger"
+    rm -f $DOKKU_ROOT/.dokkurc/DOKKU_EVENTS
+    ;;
+
+  events:list)
+    if [[ "$DOKKU_EVENTS" ]]; then
+      logged="$(find $(dirname $0) -type l -printf '%f ' | sort)"
```

Comment:
lint fails.
```
In ./plugins/20_events/commands line 31:
      logged="$(find $(dirname $0) -type l -printf '%f ' | sort)"
                     ^-- SC2046: Quote this to prevent word splitting.
```